reconcile/jira_permissions_validator.py

```diff
@@ -256,11 +256,14 @@ def validate_boards(
     return error
 
 
-def get_jira_boards(query_func: Callable) -> list[JiraBoardV1]:
+def get_jira_boards(
```

Comment:
`get_jira_boards` also called in `early_exit_desired_state`, should it also consider `jira_board_name`?

Reply:
I don't get it. What do you mean?